content/en/docs/use-cases/bootstrapping/index.md

```diff
@@ -42,23 +42,23 @@ To support the [autofill UI](/docs/reference/terms/#autofill-ui) for passkeys, m
       const available = await PublicKeyCredential.isConditionalMediationAvailable();
 
       if (available) {
-         // Query your server for options for `navigator.credentials.get()`
         try {
+          // Retrieve authentication options for `navigator.credentials.get()`
+          // from your server.
           const authOptions = await getAuthenticationOptions();
           // This call to `navigator.credentials.get()` is "set and forget."
           // The Promise will only resolve if the user successfully interacts
           // with the browser's autofill UI to select a passkey.
-          const autoFillResponse = await navigator.credentials.get({
+          const webAuthnResponse = await navigator.credentials.get({
             mediation: "conditional",
             publicKey: {
               ...authOptions,
               // see note about userVerification below
               userVerification: "preferred",
             }
           });
-           // Send the response to your server for verification.
-           // Authenticate the user if the response is valid.
```

Comment:
Why did this line get the axe? Is there a reason the RP should **not** log the user in after successful verification?

Reply:
For clarification this line was in for those (like me) who prefer to read code, so I think I'd like to see this second line remain/get reworded if need be, and essentially get duplicated in the text around the code blocks for those who don't enjoy reading code. Does that make sense?

Reply:
All right, it's just there's a lot of duplication between text and code, code and text. I suppose this is something to clean up / align eventually. In the meantime, fully understand your point and will add it again. 👍